server/events/runtime/run_step_runner.go

```diff
@@ -32,6 +33,7 @@ func (r *RunStepRunner) Run(ctx models.ProjectCommandContext, command string, pa
 		"HEAD_BRANCH_NAME":           ctx.Pull.HeadBranch,
 		"HEAD_REPO_NAME":             ctx.HeadRepo.Name,
 		"HEAD_REPO_OWNER":            ctx.HeadRepo.Owner,
+		"PATH":                       fmt.Sprintf(os.ExpandEnv("$PATH:%s"), r.TerraformBinDir),
```

Comment:
```suggestion
		"PATH":                       fmt.Sprintf("%s:%s", os.GetEnv("PATH"), r.TerraformBinDir),
```

I know this is the same but it seems more straightforward than running expandenv on the string ("$PATH:%s")